nautobot/docs/installation/http-server.md

```diff
@@ -139,11 +139,34 @@ $ sudo systemctl restart nginx
 ```
 
 !!! info
-    If the restart fails, and you changed the default key location, check to make sure the `nautobot.conf` file you pasted has the updated key location. For example, CentOS 8.2.2 requires keys to be in `/etc/pki/tls/` instead of `/etc/ssl/`.
+    If the restart fails, and you changed the default key location, check to make sure the `nautobot.conf` file you pasted has the updated key location. For example, CentOS requires keys to be in `/etc/pki/tls/` instead of `/etc/ssl/`.
 
 ## Confirm Permissions for /opt/nautobot
 
 Ensure that the `/opt/nautobot/` permissions are set to `755`.
+If permissions need to be changed, as the `nautobot` user run: `$ chmod 755 /opt/nautobot/`
```

Comment:
A couple bits of feedback here. 

- Let's please refer to this as `NAUTOBOT_ROOT` (e.g. "Confirm Permissions for `NAUTOBOT_ROOT`")
- Change the command to a code gate (triple backticks) on a new line instead of inline so it's consistent with the other places we instruct the reader to execute commands
- In the command, please use `chmod 755 $NAUTOBOT_ROOT` (again just for consistency).